### PR TITLE
style about page

### PR DESCRIPTION
**Pull request by teenie-quaggard**
Relates to #128

### CODE REVIEW

**Comment by teenie-quaggard** [comment]
> I was viewing the `/about` page and in this screenshot
> image
> It seems like there's too much white space and inconsistent styling going on. What do you think @teenie-quaggard ?

@orjwan98 happy to change! Do you mean inconsistent with the landing page? Send me your thoughts :)

File: client/src/Components/About/index.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import React, { Fragment } from 'react';
+import { Sheader, Ssection, Sp } from './About.style'
 
 const About = () => {
   return (
     <Fragment>
```
Comment (Oliversw) [inline client/src/Components/About/index.js]: Tiny detail, but is a Fragment needed if the whole thing is already contained in the Main?

**Comment by orjwan98** [comment]
@teenie-quaggard happy to merge right now if you don't wanna worry about styling too much.

File: client/src/Components/About/index.js (hunk 1 above)
Comment (teenie-quaggard) [reply client/src/Components/About/index.js]: @Oliversw nope! but we still do need to put the footer in, which will go outside of the main, but inside the fragment :)

Comment (teenie-quaggard) [reply client/src/Components/About/index.js]: > Tiny detail, but is a Fragment needed if the whole thing is already contained in the Main?

@Oliversw  - oops, i totally forgot that there is NO footer on this page! Good shout, I've removed it now :)

**Comment by teenie-quaggard** [comment]
> @teenie-quaggard happy to merge right now if you don't wanna worry about styling too much.

@orjwan98 yes, maybe it's a good idea?
It feels a bit silly to be wasting time on this. That being said, I just updated the page so that it uses the same as the landing page? What do you think? We could the little caps font variant for info pages and then regular roboto on archive entries (regular casing is easier to read for long texts).

Let me know what you think, I've been going crazy doing variants. 🐙

**Comment by orjwan98** [comment]
@teenie-quaggard I just checked it out and I love what you did. It looks much neater and more elegant. I am onboard with your suggestion too :D Ready to merge!

**Pull request merged**